federated_learning/nvflare/README.md

```
@@ -1,4 +1,4 @@
 # Federated learning with NVIDIA FLARE
 [NVFlare](https://pypi.org/project/nvflare/) provides examples on how to train federated learning models using MONAI's [FL components](https://docs.monai.io/en/latest/fl.html).
 
-## Please visit [here](https://github.com/NVIDIA/NVFlare/tree/dev/integration/monai/examples).
+## Please visit [here](https://github.com/NVIDIA/NVFlare/tree/main/integration/monai).
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)